app/helpers/pagelets_helper.rb

```diff
@@ -0,0 +1,35 @@
+module PageletsHelper
+  def pagelets_for(mountpoint)
+    Pagelets::Manager.sorted_pagelets_at("#{controller_name}/#{action_name}", mountpoint)
+  end
+
+  def render_pagelets_for(mountpoint, opts = {})
+    result = ""
+    pagelets_for(mountpoint).each do |pagelet|
+      result += render_pagelet(pagelet, opts)
+    end
+    result.html_safe
+  end
+
+  def render_tab_content_for(mountpoint, opts = {})
+    result = ""
+    pagelets_for(mountpoint).each do |pagelet|
+      result += "<div id='#{pagelet.snake_name}' class='tab-pane'>"
```

Comment:
Perhaps some kind of prefix could make it more safe, we just need an unique name. So `pagelet-name` could be unique enough.